src/app/cras/edit-cra/edit-cra.component.html

```diff
@@ -0,0 +1,114 @@
+<div class="row justify-content-center">
+  <div class="text-center">
+      <h2>{{ title[mode] }} une mission</h2>
+  </div>
+</div>
+
+<hr>
+
+<div class="row mt-5 justify-content-center">
+
+  <form class="col-12">
+
+    <div class="offset-1 col-10 mt-5">
+
+      <h4>Consultant</h4>
+
+      <hr align="left" width="70%">
+
+      <div class="form-group row justify-content-between">
+        <label for="consultantNameInput" class="col-2 col-form-label">Nom</label>
+        <div class="col-6">
+          <input
+            type="text"
+            name="consultantNameInput"
+            class="form-control"
+            id="consultantNameInput"
+            aria-describedby="consultantNameHelp"
+            [(ngModel)]="cra.consultant.name"
+            [disabled]="mode === 2"
```

Comment:
Why using an integer for this "mode" variable? A string would be more straightforward.